local-volume/provisioner/pkg/common/common.go

```diff
@@ -80,6 +85,12 @@ type UserConfig struct {
 	NodeLabelsForPV []string
 	// UseAlphaAPI shows if we need to use alpha API
 	UseAlphaAPI bool
+	// UseJobForCleaning indicates if Jobs should be spawned for cleaning block devices (as opposed to process),.
+	UseJobForCleaning bool
+	// Namespace of this Pod (optional)
+	Namespace string
```

Comment:
Can the Job namespace be different from provisioner namespace?

Reply:
The "Namespace" field here is meant to be set via the downward API in the pod manifest to be the namespace of the provisioner. This is also what the helm templates set it to. The admin could go in and explicitly change the provisioner manifest to point to some other namespace if they really want to.